wiki/Bancho_(server)/en.md

```diff
@@ -1,11 +1,26 @@
 ---
-stub: true
 tags:
   - infrastructure
 ---
 
-# Bancho (game server)
+# Bancho (server)
 
 *For other uses, see [Bancho (disambiguation)](/wiki/Disambiguation/Bancho).*
 
-**osu!bancho** is the server which handles user presence, chat and multiplayer services.
+**Bancho** (previously *osu!bancho*) is a code name of the server component which handles interactions between players running stable versions of osu!:
```

Comment:
Weird, I don't remember renaming it but I guess I did in the client in a hope to stop thinking it was what we called the "official server" 🙄

Reply:
```suggestion
**Bancho** (previously *osu!bancho*) is a code name of the server component which handles real-time interactions between players running stable versions of osu!:
```

Reply:
I rechecked and it's called `osu!Bancho` in the chat welcome message, so you didn't, correct